### DISCUSSION TITLE
Create Structure Dialog Box

### DISCUSSION THREAD

**Discussion by iGr33k** [Vector35/binaryninja-api#4306] [Q&A] [answered]
Might be a stupid question, but where is the "Create Structure" dialog box, described in the [manual](https://docs.binary.ninja/guide/cpp.html)? Also, please advise, if GitHub discussion is not the place to ask questions regarding UI.  

Mine looks like this:
![image](https://github.com/Vector35/binaryninja-api/assets/36838572/322a0dbb-0c5c-4e44-b519-7d653a37115f)  

Instead of this:  
![image](https://github.com/Vector35/binaryninja-api/assets/36838572/ff00012d-f669-46dc-814d-0a36b2f423a9)

**Comment by plafosse** [comment, marked as answer]
You need to be on the latest 3.4 version of the product to see the new structure dialog.

**Comment by plafosse** [comment]
Oh and github discussions is the perfect place to ask this question 😀
